src/charm.py

```diff
@@ -80,8 +80,8 @@ def _render_config(
     return content
 
 
-class NRFOperatorCharm(CharmBase):
-    """Main class to describe juju event handling for the SD-Core NRF operator."""
+class NRFK8sOperatorCharm(CharmBase):
```

Comment:
I think there's no need to change the class name.

Reply:
OK, Done